### ISSUE TITLE
exec /manager: exec format error on arm64 k8s

### ISSUE THREAD

**Issue by romicharroyo** [rabbitmq/messaging-topology-operator#599]
## Describe the bug
Messaging topology operator pod fails to run under arm64 Kubernetes with error `exec /manager: exec format error`

## Version and environment information
- Messaging Topology Operator: 1.10.2
- Kubernetes: 1.23
- Cloud provider or hardware configuration: AWS, c6g.2xlarge instance type (arm64 CPU)

**Comment by romicharroyo** [comment]
Yes. I also ran `docker.io/rabbitmqoperator/messaging-topology-operator:latest` on my Mac M1
```
docker run -it docker.io/rabbitmqoperator/messaging-topology-operator:latest                                                            
{"level":"info","ts":"2023-04-19T13:31:32Z","logger":"setup","msg":"unable to find operator namespace"}
```

But. In doesn't work on AWS (EKS) with AWS Graviton CPU.

<img width="773" alt="Screenshot 2023-04-19 at 16 35 48" src="https://user-images.githubusercontent.com/50058809/233091952-cabd0f5b-0bd9-4aa7-8c14-cfc290ca1cfe.png">

**Comment by MirahImage** [comment]
Hmmm, that is indeed puzzling.

**Comment by MirahImage** [comment]
Can you attach a debug container to the pod and let us know what architecture is reported when running `uname -m`? 
https://kubernetes.io/docs/tasks/debug/debug-application/debug-running-pod/#ephemeral-container

**Comment by romicharroyo** [comment]
On my k8s cluster on AWS arch is `aarch64`

**Comment by DanielePalaia** [comment]
Maybe this discussion can help ? https://github.com/rabbitmq/messaging-topology-operator/issues/570
There was a fix but a new version of the operator wasn't yet released. Can you try with the image specified on the thread?

**Comment by MirahImage** [comment]
We've released a 1.10.3 patch which should hopefully have a working multi-arch image 🤞🏼
